DetourCrowd/Source/DetourObstacleAvoidance.cpp

```diff
@@ -331,7 +331,7 @@ float dtObstacleAvoidanceQuery::processSample(const float* vcand, const float cs
 	// find the threshold hit time to bail out based on the early out penalty
 	// (see how the penalty is calculated below to understnad)
 	float minPen = minPenalty - vpen - vcpen;
-	float tThresold = ((double)m_params.weightToi/(double)minPen - 0.1) * (double)m_params.horizTime;
+	float tThresold = (m_params.weightToi / minPen - 0.1f) * m_params.horizTime;
```

Comment:
I'm not sure why these were being cast to double before the division. Maybe it was done in an attempt to increase the precision of the division?  Unless there's something I'm missing, this is generally not the case for IEEE 754 floats.